src/adaptation/icap/ModXact.cc

```diff
@@ -1157,6 +1157,31 @@ void Adaptation::Icap::ModXact::decideOnParsingBody()
     }
 }
 
+namespace Adaptation {
+namespace Icap {
+class ExtensionsParser : public Http1::CustomExtensionsParser
+{
+public:
+    ExtensionsParser() : useOriginBody_(-1) {}
+
+    virtual bool parse(Tokenizer &tok, const SBuf &extName) override {
+        assert(knownExtension(extName));
+        return parseIntExtension(tok, UseOriginalBodyName, useOriginBody_);
+    }
+    virtual bool knownExtension(const SBuf &extName) override { return extName == UseOriginalBodyName;}
+    int64_t useOriginBody() const { return useOriginBody_; }
+
+private:
+    static const SBuf UseOriginalBodyName;
+
+    int64_t useOriginBody_;
```

Comment:
No need to propagate the old naming mistake:

```suggestion
    int64_t useOriginalBody_ = -1;
```